src/components/validation-report/styled.ts

```diff
@@ -39,6 +40,12 @@ const InfoIcon = styled(InfoSVG)`
   }
 `;
 
+const SummaryLink = styled(ScrollLink)`
+  display: flex;
+  flex: 1;
+  padding: ${theme.spacing('S6')};
+`;
+
 const SummaryBox = styled.div`
```

Comment:
Hvorfor ikke bare bruke denne som scroll link base?
```suggestion
const SummaryBox = styled(ScrollLink)`
```

Reply:
Siden det er en box og ikke en lenke.

Reply:
Siden det er en Box og ikke en lenke. I tillegg så trenger ikke ConformsSummary en lenke. ScrollLink to property er obligatorisk.